completions/bash/oci-runtime-tool

```diff
@@ -119,6 +119,13 @@ __oci-runtime-tool_complete_log_level() {
 	" -- "$cur" ) )
 }
 
+__oci-runtime-tool_complete_compliance_level() {
+	COMPREPLY=( $( compgen -W "
+		must
+		should
```

Comment:
Missing `may` (which is documented [here][1]).  Although I'm not sure what MAY-level validation looks like, and would be fine dropping it from `--compliance-level`.

[1]: https://github.com/opencontainers/runtime-tools/pull/354/files#diff-6c88621051d8dcd99407582c2b76fd68R713